rest_api/network_apis/service-core-v1.adoc

```diff
@@ -116,7 +116,10 @@ This field may hold a maximum of two entries (dual-stack families, in either ord
 
 | `loadBalancerSourceRanges`
 | `array (string)`
-| If specified and supported by the platform, this will restrict traffic through the cloud-provider load-balancer will be restricted to the specified client IPs. This field will be ignored if the cloud-provider does not support the feature." More info: https://kubernetes.io/docs/tasks/access-application-cluster/configure-cloud-provider-firewall/
+| If specified and supported by the platform, this restricts traffic through the load-balancer of the cloud-provider to the specified client IPs. This field will be ignored if the cloud-provider does not support the feature. For more information, see https://kubernetes.io/docs/tasks/access-application-cluster/configure-cloud-provider-firewall/
```

Comment:
@chinmayi-chandrasekar, all of this is automatically generated. I'm not sure we should be modifying this here rather than upstream in the Kubernetes project itself.

Reply:
I have undone changes to this file.